### PR TITLE
Set DNS server explicitly while creating DHCP

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
Upgrade power-go client

**What this PR does / why we need it**:
Pass DNS server explicitly while creating DHCP.

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/infra/powervs/create.go [hunk 1]
```diff
@@ -1114,7 +1114,7 @@ func (infra *Infra) setupPowerVSDHCP(options *CreateInfraOptions, session *ibmpi
 // createPowerVSDhcp creates a new dhcp server in powervs
 func (infra *Infra) createPowerVSDhcp(options *CreateInfraOptions, client *instance.IBMPIDhcpClient) (dhcpServer *models.DHCPServerDetail, err error) {
 	startTime := time.Now()
-	dhcp, err := client.Create(&models.DHCPServerCreate{CloudConnectionID: &infra.PowerVSCloudConnectionID})
+	dhcp, err := client.Create(&models.DHCPServerCreate{CloudConnectionID: &infra.PowerVSCloudConnectionID, DNSServer: utilpointer.String("1.1.1.1")})
```
Comment (enxebre) [inline cmd/infra/powervs/create.go:1117]: can you provide some context and add a comment on why is this needed and where is it coming from?

Comment (dharaneeshvrd) [reply cmd/infra/powervs/create.go:1118]: Added a comment.

**Review by alvaroaleman — APPROVED** [review]
